lib/cn/kvset.c

```diff
@@ -580,17 +518,17 @@ kvset_open2(
             kcachesz += kblk->kb_klen_max + kblk->kb_klen_min;
 
         /* kvset_stats from kblocks */
-        ks->ks_st.kst_kalen += props.mpr_alloc_cap;
-        ks->ks_st.kst_kwlen += props.mpr_write_len;
+        ks->ks_st.kst_halen += kblk->kb_kblk_desc.alen_pages * PAGE_SIZE;
+        ks->ks_st.kst_hwlen += kblk->kb_kblk_desc.wlen_pages * PAGE_SIZE;
```

Comment:
Hmm, I think we're missing ptombs from this accumulation?

Reply:
Kblocks don't have ptombs anymore.  They're in the hblock.